go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/onsi/gomega v1.33.1
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
 	github.com/stretchr/testify v1.8.4
-	go.anx.io/go-anxcloud v0.7.0
+	go.anx.io/go-anxcloud v0.7.1-0.20240531094418-52eb9501b9ab
```

Comment:
As soon as https://github.com/anexia-it/go-anxcloud/pull/379 is released, this should be updated again. ^^